ui/pages/mobile-sync/mobile-sync.component.js

```diff
@@ -338,7 +338,7 @@ export default class MobileSyncPage extends Component {
         <label className="input-label" htmlFor="password-box">
           {t('enterPasswordContinue')}
         </label>
-        <div className="input-group">
+        <div>
```

Comment:
This is an update from me. The className isn't used anywhere: 

![Screenshot 2023-02-05 at 8 26 14 PM](https://user-images.githubusercontent.com/8112138/216883556-d1d5f528-5177-4e10-bfcc-3dfc221c3cb5.png)